crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1074,6 +1075,32 @@ impl World {
         QueryState::new(self)
     }
 
+    /// Add a [`With<T>`](crate::prelude::With) filter to [`DefaultQueryFilters`].
+    /// Inserts the [`DefaultQueryFilters`] resource if it does not exist
+    pub fn default_with_filter<T: Component>(&mut self) {
```

Comment:
I think this would be clearer as "set_default_with_filter" (and the same below).